tests/ReflectionClassTest.php

```diff
@@ -47,21 +48,21 @@ public function testGetModifiers($fileName)
     /**
      * Performs method-by-method comparison with original reflection
      *
-     * @dataProvider caseProvider
+     * @dataProvider methodCaseProvider
      *
-     * @param ReflectionClass   $parsedClass Parsed class
-     * @param \ReflectionMethod $refMethod Method to analyze
-     * @param string                  $getterName Name of the reflection method to test
+     * @param \ReflectionClass $refClass Original reflection class
+     * @param ReflectionClass $parsedClass Parsed class
+     * @param string $getterName Name of the reflection method to test
      */
     public function testReflectionMethodParity(
+        \ReflectionClass $refClass,
```

Comment:
I guess `methodCaseProvider` now returns more data, but this particular test doesn't use that extra data?